decidim-core/spec/lib/shakapacker/configuration_spec.rb

```diff
@@ -24,7 +24,7 @@ module Shakapacker
         let!(:config_file) { subject.configuration_file }
 
         it "returns the runtime configuration path" do
-          expect(config_file).to eq(runtime_config_path.to_s)
+          expect(config_file).to eq(Rails.application.root.join("tmp/shakapacker_runtime#{ENV.fetch("TEST_ENV_NUMBER", "")}.yml").to_s)
```

Comment:
```suggestion
          expect(config_file).to eq(runtime_config_path.to_s)
```

Reply:
Let's move this change to the actual assignment of `runtime_config_path` on line 17